.github/CODEOWNERS

```diff
@@ -1,116 +1,55 @@
 * @bader
 
 # Front-end compiler
-clang/ @premanandrao @elizabethandrews @smanna12
+clang/ @intel/dpcpp-cfe-reviewers
 
-# Driver
-clang/**/Driver @mdtoguchi @AGindinson @hchilama
+# Clang driver
+clang/**/Driver @intel/dpcpp-clang-driver-reviewers
 
 # LLVM-SPIRV translator
-llvm-spirv/ @AlexeySotkin @AlexeySachkov
+llvm-spirv/ @intel/dpcpp-spirv-reviewers
 
 # OpenCL deps
-opencl/ @pvchupin @vladimirlaz
+opencl/ @intel/llvm-reviewers-runtime
 
 # OpenCL "offline" compiler
-opencl/opencl-aot/ @dm-vodopyanov @AlexeySachkov @romanovvlad
+opencl/opencl-aot/ @intel/dpcpp-tools-reviewers
 
 # Device library
-libdevice/ @vzakhari @jinge90
+libdevice/ @intel/llvm-reviewers-runtime
 
 # DPC++ runtime library
 sycl/ @intel/llvm-reviewers-runtime
 
 # Documentation
-sycl/ReleaseNotes.md @pvchupin @tfzhu
-sycl/doc/ @pvchupin @bader
+sycl/ReleaseNotes.md @intel/dpcpp-doc-reviewers
+sycl/doc/ @intel/dpcpp-doc-reviewers
+sycl/doc/design/ @intel/dpcpp-specification-reviewers
 sycl/doc/extensions/ @intel/dpcpp-specification-reviewers
-sycl/doc/extensions/SPIRV/ @AlexeySotkin @bashbaug @mbelicki
+sycl/doc/extensions/SPIRV/ @intel/dpcpp-spirv-doc-reviewers
 sycl/doc/dev @bader @vladimirlaz
 
-# Sub-groups
-sycl/include/CL/sycl/detail/spirv.hpp @Pennycook @AlexeySachkov
-sycl/include/sycl/ext/intel/group_algorithm.hpp @Pennycook @AlexeySachkov
-sycl/include/sycl/ext/intel/sub_group.hpp @Pennycook @AlexeySachkov
-sycl/include/sycl/ext/intel/sub_group_host.hpp @Pennycook @AlexeySachkov
-
-# PI API
-sycl/include/CL/sycl/detail/pi.def @smaslov-intel
-sycl/include/CL/sycl/detail/pi.h   @smaslov-intel
-sycl/include/CL/sycl/detail/pi.hpp @smaslov-intel
-sycl/include/CL/sycl/detail/pi* @smaslov-intel
-sycl/plugins/ @smaslov-intel
-sycl/source/detail/pi.cpp @smaslov-intel
-sycl/source/detail/plugin.hpp @smaslov-intel
-sycl/source/detail/posix_pi.cpp @smaslov-intel
-sycl/source/detail/windows_pi.cpp @smaslov-intel
+# Level Zero plugin
+sycl/sycl/plugins/level_zero @intel/dpcpp-l0-pi-reviewers
```

Comment:
@vladimirlaz, code review assignment is broken.
I think it's due to bugs in the .github/CODEOWNERS file.

sycl/sycl/plugins/level_zero -> sycl/plugins/level_zero?